assets/src/components/ImageOrButton/ImageOrButton.js

```diff
@@ -1,56 +1,57 @@
-import {Component} from '@wordpress/element';
 import {Button} from '@wordpress/components';
 import {MediaUpload,MediaUploadCheck} from '@wordpress/block-editor';
 
-export class ImageOrButton extends Component {
-  constructor(props) {
-    super(props);
-  }
-
-  render() {
-    const {__} = wp.i18n;
-    const disabled = this.props.disabled ?? false;
-
-    const getImageOrButton = (openEvent) => {
-      if ( this.props.imageId ) {
-
-        return (
-
-          <img
-            src={ this.props.imageUrl }
-            onClick={ openEvent }
-            className={ this.props.imgClass }
-          />
-
-        );
-      }
-      else {
-        return (
-          <div className='button-container'>
-            <Button
-              onClick={ openEvent }
-              className='button'
-              disabled={ disabled }>
-              { this.props.buttonLabel }
-            </Button>
-
-            <div>{ this.props.help }</div>
-          </div>
-        );
-      }
-    };
-
-    return <div className='ImageOrButton'>
-      <MediaUploadCheck>
-        <MediaUpload
-          title={this.props.title}
-          type='image'
-          onSelect={this.props.onSelectImage}
-          value={this.props.imageId}
-          allowedTypes={['image']}
-          render={ ({ open }) => getImageOrButton(open) }
+export const ImageOrButton = props => {
+  const {__} = wp.i18n;
+  const {
+    disabled = false,
+    imageId,
+    imageUrl,
+    imgClass,
+    buttonLabel,
+    help,
+    title,
+    onSelectImage,
+  } = props;
+
+  const getImageOrButton = (openEvent) => {
+    if (imageId) {
+
+      return (
+        <img
+          src={ imageUrl }
+          style={ { border: '1px solid grey', minWidth: '100px', minHeight: '100px', } }
```

Comment:
Ah left this one in, but it's probably something we should tackle. If the ID doesn't exist, the image currently is not visible or clickable. We should probably catch this case and show an appropriate error message.